api/config/config_local.json

```diff
@@ -85,7 +85,8 @@
     "unsafeAllowDeleteUser": true,
     "enableVpcFlowLogs": true,
     "enableVpcServicePerimeter": true,
-    "enableNewAccountCreation": true
+    "enableNewAccountCreation": true,
+    "enableBillingLockout": true
```

Comment:
nit: I think we should keep local and test in sync typically. I don't feel strongly about both true or both false

Reply:
aside: is it too late to rename `enableNewAccountCreation`? It's really talking about a revised flow, not enabling people to create new accounts altogether?

Reply:
sounds good.